guide/contribute/content-guidelines.md

```diff
@@ -33,13 +33,17 @@ This guide is crafted for anyone interested in building an effective Bitcoin app
 
 Address the reader directly. Whenever possible, try using active instead of passive voice. Make the reader a part the conversation by using second-person pronouns like "you, your and yours". Do not write in first-person and avoid giving predictions and personal opinions.
 
-##### Do
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```

Comment:
You can also consider definition lists for these ([see here](https://bitcoin.design/guide/contribute/formatting/#definition-list)). Those are great for a before/after representation of text.

Reply:
Will play around with that and see how it looks :) cheers!